src/aux/operations_ids.py

```diff
@@ -14,6 +14,7 @@ class OPERATION(Enum):
     SUBSCRIPTION = 4
     UE_PATH_LOSS = 5
     SERVING_CELL_INFO = 6
+    HANDOVER = 7
```

Comment:
These two Enums cannot have the same value.
Please update this accordingly. Keep in mind that if you update the value of the `E2E_UE_PERFORMANCE` enum, you will also have to update the test that invokes that endpoint.

Reply:
fixed!